tools/tools.go

```diff
@@ -0,0 +1,7 @@
+// +build tools
+
+package tools
+
+import (
+	_ "github.com/go-bindata/go-bindata"
```

Comment:
Is this to make sure go module pulls this in? If so, might want to write a small comment explaining such...

Reply:
Fixed.